inspire_disambiguation/api.py

```diff
@@ -71,7 +73,8 @@ def train_and_save_distance_model(
     curated_signatures = get_signatures(only_curated=True)
```

Comment:
maybe add runtime also for the training?

Reply:
:+1: